.github/workflows/update_website.yml

```diff
@@ -33,57 +29,26 @@ jobs:
         run: |
           ls -a
```

Comment:
Why this step?

Reply:
This was probably for debugging, removed.